spec/type_aliases/backend_url_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+describe 'Sensu::Backend_URL' do
+  context 'allows valid values' do
+    [
+      'localhost:8081',
+      '127.0.0.1:8081',
+      'ws://localhost:8081',
+      'wss://localhost:8081',
+      'ws://127.0.0.1:8081',
+      'wss://127.0.0.1:8081',
+      'test.example.com:8081',
+      'ws://test.example.com:8081',
+      'wss://test.example.com:8081',
+    ].each do |value|
+      describe value.inspect do
+        it { is_expected.to allow_value(value) }
+      end
+    end
+  end
+
+  context 'disallow invalid values' do
+    [
+      'http://localhost:8081',
```

Comment:
you can't have a HTTP backend?

Reply:
Not that I'm aware of, docs say use `ws://` or `wss://` (ssl).

Reply:
Confirmed with vagrant, have to use `ws://` or `wss://`, tried using `https://`:

```
Jan  3 01:02:51 el7-agent sensu-agent: {"component":"agent","error":"malformed ws or wss URL","level":"error","msg":"reconnection attempt failed","time":"2019-01-03T01:02:51Z"}
```